duckduckgo/Android
# Feature/bookmark favicons

Task/Issue URL: https://app.asana.com/0/547792610048271/532522547697235
Tech Design URL: 
CC: 

**Description**:
Add favicon support for bookmark screen

**Steps to test this PR**:
1. Launch bookmark screen and ensure bookmarks load a favicon icon
1. Edit a bookmark (ex: change from facebook.com to twitter.com) and ensure favicon updates
1. Edit a bookmark to contain invalid url (eg: `https:/asdf`) and ensure the globe is used as a placeholder


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
